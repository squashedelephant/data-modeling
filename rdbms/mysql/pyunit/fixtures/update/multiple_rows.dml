INSERT INTO data(name) VALUES ('Alpha'), ('Beta'), ('Gamma'), ('Delta'), ('Epsilon');
UPDATE data SET name='Chi' WHERE name='Gamma';
UPDATE data SET name='Upsilon' WHERE name='Epsilon';
