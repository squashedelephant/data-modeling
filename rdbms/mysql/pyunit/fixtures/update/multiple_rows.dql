SELECT id FROM data WHERE name IN ('Chi', 'Upsilon');
