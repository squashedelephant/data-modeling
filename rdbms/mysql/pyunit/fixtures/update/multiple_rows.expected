3L
5L
